test/Google_Api_Test/src/Main.enso

```diff
@@ -10,19 +12,11 @@ main filter=Nothing =
 
 
 add_specs suite_builder =
-    suite_builder.group "Google Spreadsheets" pending="outdated" group_builder->
-        group_builder.specify "should allow downloading a spreadsheet" <|
-            secret = enso_project.data / 'secret.json'
-            api = Google_Sheets.initialize secret
-            sheet_id = '1WjVQhYdc04RwdWB22RNLgfQiLeWYhxiij1_xj22RDq0'
-            sheet_range = 'Sheet1!A1:B6'
-            table = api.spreadsheets.get_table sheet_id sheet_range
-            table.columns.map .name . should_equal ["Foo", "Bar"]
-            table.at 'Foo' . to_vector . should_equal [1,2,3,4,5]
-            table.at 'Bar' . to_vector . should_equal ["hello", "world", "foo", "bar", "baz"]
-
-    suite_builder.group "Google Analytics" group_builder->
-        group_builder.specify "should allow downloading a spreadsheet" <|
-            secret = enso_project.data / 'secret.json'
-            accounts = Google_Analytics.list_accounts secret
-            accounts.length . should_equal 1
+    suite_builder.group "Google Sheets" group_builder->
+        group_builder.specify "can read a workbook via Google_Sheets.read" <|
+            r = Google_Sheets.read '1ifGXhOlOp2xn4l9bGYbqGvAoSsG5M_M2ga60WFaBZwg' (Enso_Secret.get 'GS')
+            r.sheet_names.should_equal ['Sheet1']
+        group_builder.specify "can read a worksheet via Google_Sheets_Workbook.read" <|
+            r = Google_Sheets.read '1ifGXhOlOp2xn4l9bGYbqGvAoSsG5M_M2ga60WFaBZwg' (Enso_Secret.get 'GS')
+            r.sheet_names.should_equal ['Sheet1']
+            r.read 'Sheet1' . should_equal (Table.new [['X', ['Y', 'Z']]])
```

Comment:
Shouldn't these tests use `Cloud_Tests_Setup` and have `pending=setup.real_cloud_pending`?

All of our tests that rely on the Cloud do.

Moreover, where does the `GS` secret come from? I imagine you have set it up on the GitHub account? How would I set it up on a personal account if I wanted to run these tests? Probably would be good to document it somewhere.

Reply:
Yep. I will document how to run the test suite as part of the overall feature. I think with the OAuth token it will be hard to run on CI. For now I am imagining devs will have to run locally.

Reply:
That's why I was curious that there's no `pending` condition.

IIRC all `test/*_Tests` are run on CI, so if there's no pending nor secret set up on the CI account, this test would be failing and blocking your PR from integration.

Is some other condition making this test suite not run?